lib/astarte/core/generators/interface.ex

```diff
@@ -24,17 +24,18 @@ defmodule Astarte.Core.Generators.Interface do
   """
   use ExUnitProperties
 
-  alias Ecto.UUID
-  alias Astarte.Core.Interface
   alias Astarte.Core.Generators.Mapping, as: MappingGenerator
+  alias Astarte.Core.Interface
+  alias Ecto.UUID
 
   @doc """
   Generates a valid Astarte Interface.
 
   https://github.com/astarte-platform/astarte_core/blob/master/lib/astarte_core/interface.ex
+  TODO: Replace map() with the actual Astarte.Core.Interface type when available
   """
-  @spec interface() :: StreamData.t(%Interface{})
-  def interface() do
+  @spec interface() :: StreamData.t(map())
```

Comment:
we should have `t` types defined for our data. ref: https://github.com/astarte-platform/astarte_core/pull/116
we may need to forward port it though

```suggestion
  @spec interface() :: StreamData.t(Interface.t())
```